gtest-parallel

```diff
@@ -169,8 +181,8 @@ class FilterFormat:
       self.total_tests = int(arg.split(' ', 1)[1])
       self.out.transient_line("[0/%d] Running tests..." % self.total_tests)
 
-  def logfile(self, job_id, name):
-    self.outputs[job_id] = name
+  def logfile(self, job_id, file_name):
```

Comment:
If this one is not needed here, can you work it away for the RawFormat class too and get rid of the call here?